src/com/palmergames/bukkit/towny/object/metadata/MetaMap.java

```diff
@@ -0,0 +1,16 @@
+package com.palmergames.bukkit.towny.object.metadata;
+
+import java.util.HashMap;
+
+public class MetaMap extends HashMap<String, CustomDataField<Object>> {
```

Comment:
I don't see the necessary to extend the hashmap class. The `keySet()` method is set to public and this could be easily put into a static method maybe into the customdatafield class. Only reason I can think of is for condensed readability, but that doesn't seem like the most valid justification.

Reply:
Yeah it makes more sense to just use regular HashMap